### DISCUSSION TITLE
Remove embedded jpg album image

### DISCUSSION THREAD

**Discussion by nitanmarcel** [libsndfile/libsndfile#875] [Q&A]
Is there a way to use this lib and c++ to remove the artwork from an mp3 audio file?

I see there is a function to set and get mtadata, but only for string values.

Thanks.

**Comment by evpobr** [comment]
Hi @nitanmarcel . Unfortunately we don't have that feature.

**Reply by nitanmarcel** [reply]
Hello. Thanks.

~~Could you take a look the the issue I've open? I still don't know if this is how it's supposed to be or not: https://github.com/libsndfile/libsndfile/issues/876~~ 

Ah I see you labeled the issue. I also added a sample mp3 a couple of seconds ago :)

**Comment by evpobr** [comment]
Thanks, i hope we will find time to fix it.

**Reply by nitanmarcel** [reply]
In the meantime, any idea of what workaround I could try to implement? I'm totally stuck here.

Thanks

**Reply by evpobr** [reply]
Hmm, may @arthurt may help.
